lib/fluent/log.rb

```diff
@@ -96,8 +96,12 @@ def initialize(logger, opts={})
       @level = logger.level + 1
       @debug_mode = false
       @log_event_enabled = false
-      @time_format = '%Y-%m-%d %H:%M:%S %z '
       @depth_offset = 1
+      @format = nil
+      @time_format = nil
+      @formatter = nil
+
+      self.format = :text
```

Comment:
Need to set other default values: https://github.com/fluent/fluentd/pull/1644/files#diff-d845866477e75d27b77c4b8acc63c7d9R163